File: app/src/main/java/com/bottle/track/user/BaseResponseBean.java
package com.bottle.track.user;

/**
 * Created by Administrator on 2017/12/12 0012.
 */

public class BaseResponseBean<T> {

    private int rescode;
    private T result;
    private String info;

    public int getRescode() {
        return rescode;
    }

    public void setRescode() {
        this.rescode = rescode;
    }

    public T getResult() {
        return result;
    }

    public void setResult(T result) {
        this.result = result;
    }

    public String getInfo() {
        return info;
    }

    public void setInfo() {
        this.info = info;
    }
}

File: app/src/main/java/com/bottle/track/api/BaseResponseBean.kt
package com.bottle.track.api

/**
 * @Date 2017/11/21 11:02
 * @Author halfbottle
 * @Version 1.0.0
 * @Description {"rescode":true,"info":"success","result":{"keya":"Android","keyb":"iOS"}}
 */
class BaseResponseBean<T>{

    var rescode: Int? = null
    private set

    var info: String? = null
    private set

    var result: T? = null
    private set

}